docs/views/pages/ExamplesPage.js

```diff
@@ -30,6 +30,7 @@ class Examples extends Component {
       infinite: true,
       indicators: true,
       pauseOnHover: true,
+      dir: 'ltr',
```

Comment:
Is this needed?

Reply:
It has been removed in the latest commit